packages/cli/test/unit/commands/pg/promote.unit.test.ts

```diff
@@ -0,0 +1,855 @@
+import {stderr} from 'stdout-stderr'
+import Cmd from '../../../../src/commands/pg/promote'
+import runCommand from '../../../helpers/runCommand'
+import expectOutput from '../../../helpers/utils/expectOutput'
+import {expect} from 'chai'
+import * as nock from 'nock'
+import heredoc from 'tsheredoc'
+import * as fixtures from '../../../fixtures/addons/fixtures'
+const stripAnsi = require('strip-ansi')
+
+describe('pg:promote when argument is database', () => {
+  const addon = fixtures.addons['dwh-db']
+  const pgbouncerAddonID = 'c667bce0-3238-4202-8550-e1dc323a02a2'
+
+  beforeEach(() => {
+    nock('https://api.heroku.com')
+      .post('/actions/addon-attachments/resolve')
+      .reply(200, [{addon}])
+    nock('https://api.heroku.com')
```

Comment:
You can safely remove this line and chain the following `get` request, as it's hitting the same domain.

Reply:
Mmm... It was only meant for line 19, 😆 .